From 6ed95c988414580ef3f581c3bd467a49ab1a1af2 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Tue, 16 Dec 2025 15:28:08 +0100
Subject: [PATCH 089/211] EDITME: downstream-cameras

These should be most likely replaced by using CCS driver which is
generic and is inside the kernel.

The CCS driver may need more work (and implement quirks) before this
happens.

There already has been done work for IMX519, which is used by
RaspberryPi extension boards.

# Describe the purpose of this series. The information you put here
# will be used by the project maintainer to make a decision whether
# your patches should be reviewed, and in what priority order. Please be
# very detailed and link to any relevant discussions or sites that the
# maintainer can review to better understand your proposed changes. If you
# only have a single patch in your series, the contents of the cover
# letter will be appended to the "under-the-cut" portion of the patch.

# Lines starting with # will be removed from the cover letter. You can
# use them to add notes or reminders to yourself. If you want to use
# markdown headers in your cover letter, start the line with ">#".

# You can add trailers to the cover letter. Any email addresses found in
# these trailers will be added to the addresses specified/generated
# during the b4 send stage. You can also run "b4 prep --auto-to-cc" to
# auto-populate the To: and Cc: trailers based on the code being
# modified.

Signed-off-by: David Heidelberg <david@ixit.cz>

--- b4-submit-tracking ---
# This section is used internally by b4 prep for tracking purposes.
{
  "series": {
    "revision": 1,
    "change-id": "20251216-downstream-cameras-2af0cfa9933f",
    "prefixes": []
  }
}
-- 
2.55.0

